runelite-client/src/main/java/net/runelite/client/plugins/stretchedmode/StretchedModeConfig.java

```diff
@@ -23,19 +23,20 @@
  * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package net.runelite.client.plugins.stretchedfixedmode;
+package net.runelite.client.plugins.stretchedmode;
 
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
 
-@ConfigGroup("stretchedfixedmode")
-public interface StretchedFixedModeConfig extends Config
+@ConfigGroup("stretchedmode")
+public interface StretchedModeConfig extends Config
 {
 	@ConfigItem(
 		keyName = "keepAspectRatio",
 		name = "Keep aspect ratio",
-		description = "Keeps the aspect ratio when stretching"
+		description = "Keeps the aspect ratio when stretching.",
+		position = 1
```

Comment:
As I stated in my reply to @Adam- 's comment, these positions are useless as it is sorted by name already (and the position is exactly same as explicitly specified here)

Reply:
General stretching options at the top, resizable options at the bottom. 
Promise you'll catch anyone trying to add a non-resizable related config at the bottom and I'll remove it.

Reply:
I dont think we will be adding any config here in future

Reply:
Remove positions.